develop/rendering/hud.md

```diff
@@ -1,42 +1,46 @@
 ---
 title: Rendering in the Hud
-description: Learn how to use the HudRenderCallback event to render to the hud.
+description: Learn how to use the Fabric Hud API to render to the hud.
```

Comment:
I'm not sure whether it's in the scope of this pull request, but `HUD` is an abbreviation and should be written in all caps everywhere.

Otherwise, looks fine to me.